nav2_collision_monitor/include/nav2_collision_monitor/pointcloud.hpp

```diff
@@ -98,7 +99,9 @@ class PointCloud : public Source
   // ----- Variables -----
 
   /// @brief PointCloud data subscriber
-  nav2::Subscription<sensor_msgs::msg::PointCloud2>::SharedPtr data_sub_;
+  std::shared_ptr<point_cloud_transport::PointCloudTransport> pct_;
```

Comment:
I think we need the preprocessor ifs here for rolling and newer and kilted and older so this compiles with jazzy and kilted